src/hooks/useUserLocation.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import { getDailySummary, groupByDate, sortByDate } from '../utils';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
import { getDailySummary, groupByDate, sortByDate } from "../utils";
```